kotlinpoet-metadata-specs/src/main/kotlin/com/squareup/kotlinpoet/metadata/specs/KotlinPoetMetadataSpecs.kt

```diff
@@ -816,6 +826,19 @@ private inline fun <E> setOf(body: MutableSet<E>.() -> Unit): Set<E> {
 private val OVERRIDE = Override::class.asClassName()
 private val JVM_DEFAULT = JvmDefault::class.asClassName()
 private val JVM_STATIC = JvmStatic::class.asClassName()
+private fun createThrowsSpec(
+  exceptions: Set<TypeName>,
+  useSiteTarget: UseSiteTarget? = null
+): AnnotationSpec {
+  return AnnotationSpec.builder(Throws::class)
+      .addMember(
+          "exceptionClasses = %L",
+          exceptions.map { CodeBlock.of("%T::class", it) }
```

Comment:
nit: These won't get new lines automatically if the array is too long, this would require some manual formatting. Not important for this PR.